spec/lib/task_helpers/exports/roles_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe TaskHelpers::Exports::Roles do
+  let(:role_test_export) do
+    [{"name"                => "Test Role",
+      "read_only"           => false,
+      "settings"            => nil,
+      "feature_identifiers" => ["about"]}]
+  end
+
+  let(:role_super_export) do
+    [{"name"                => "EvmRole-super_administrator",
+      "read_only"           => true,
+      "settings"            => nil,
+      "feature_identifiers" => []}]
+  end
+
+  let(:export_dir) do
+    Dir.mktmpdir('miq_exp_dir')
+  end
+
+  before do
+    FactoryGirl.create(:miq_user_role, :name => "Test Role", :features => "about")
+    FactoryGirl.create(:miq_user_role, :role => "super_administrator")
+  end
+
+  after do
+    # FileUtils.remove_entry export_dir
```

Comment:
Do we need this or can we just remove this block?

Reply:
It's needed so that when the test runs the directory that is created is removed. I had commented it out when I was testing the spec files and forgot to uncomment it before committing. I've uncommented the line.